packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -355,12 +363,34 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
         @mouseenter=${this._handleMouseEnterSelectable}
         @mouseleave=${this._handleMouseLeaveSelectable}
       >
-        ${this.renderTitle} ${this._renderActionButton()}
+        ${this._renderTitle()} ${this._renderActionButton()}
       </div>
       ${this._renderMetadata()}
     </div>`;
   }
 
+  private _handleNavigation(event: KeyboardEvent, directionMatches: boolean): void {
+    if (!this.subtleSelect || this._isCheckboxDisplayed()) {
+      return;
+    }
+
+    if (event.key == 'Tab' && directionMatches) {
+      event.preventDefault();
+      this._isSelectableHovered = true;
+      new Promise((resolve) => requestAnimationFrame(resolve)).then(() => {
```

Comment:
Goodness...what a hassle to get things working across browsers.

Reply:
An intricate dance, indeed 😄